lib/colonel_kurtz/block_type_content.ex

```diff
@@ -40,6 +40,16 @@ defmodule ColonelKurtz.BlockTypeContent do
 
       @derive [Jason.Encoder]
 
+      @spec from_map(map) :: block_content
+      def from_map(attrs) do
```

Comment:
Same deal: just a reorg, no changes. `from_map` is the "entrypoint" into this schema so I like seeing it first.